samples/Wifi_Sniffer/Makefile

```diff
@@ -6,15 +6,12 @@
 include ./Makefile-user.mk
 
 # Important parameters check.
-# We need to make sure SMING_HOME and ESP_HOME variables are set.
+# We need to make sure SMING_HOME is set.
 # You can use Makefile-user.mk in each project or use enviromental variables to set it globally.
  
 ifndef SMING_HOME
-$(error SMING_HOME is not set. Please configure it in Makefile-user.mk)
-endif
-ifndef ESP_HOME
```

Comment:
You are right that the ESP_HOME is not needed for every Arch. But I would suggest to have an optional Makefile with requirements per ARCH.  Something like

```
-include $(SMING_HOME)/$(SMING_ARCH)/requirements.mk
```

And move the ESP_HOME requirement to a the new `$(SMING_HOME)/Esp8266/requirements.mk` file.

Reply:
I see I've managed to leave `ESP_HOME` in `app.mk`, I'll remove that. It's defined in `build.mk` along with all the other arch. requirements. Don't think we need another makefile for that, do we?

Reply:
> Don't think we need another makefile for that, do we?

Wonderful! Then nothing more is needed.

Reply:
The `makefile-user.mk` is still cluttered with arch stuff, not entirely sure how to approach that so could leave it for another time?

Reply:
> he makefile-user.mk is still cluttered with arch stuff, not entirely sure how to approach that so could leave it for another time?

Yes, leave it for now.